pkgs/servers/code-server/default.nix

```diff
@@ -61,8 +64,8 @@ in stdenv.mkDerivation rec {
 
     # to get hash values use nix-build -A code-server.prefetchYarnCache
     outputHash = {
-      x86_64-linux = "1443qwkllb714s4qw3b9y1mcc6p2ykgc02pw2k3z2gczvvr0g8qv";
-      aarch64-linux = "1443qwkllb714s4qw3b9y1mcc6p2ykgc02pw2k3z2gczvvr0g8qv";
```

Comment:
What happened to aarch64?

Reply:
I had removed it since it wasn't in `meta.platforms`, I'll add it back though